cmd/lncli/commands.go

```diff
@@ -2105,3 +2108,29 @@ func updateFees(ctx *cli.Context) error {
 	printRespJSON(resp)
 	return nil
 }
+
+var setAliasCommand = cli.Command{
+	Name:      "setalias",
+	Usage:     "set the alias of this node",
+	ArgsUsage: "alias",
+	Action:    actionDecorator(setAlias),
+}
+
+func setAlias(ctx *cli.Context) error {
+	ctxb := context.Background()
+	client, cleanUp := getClient(ctx)
+	defer cleanUp()
+
+	newAlias := ctx.Args().First()
```

Comment:
Here you could use the patter that is used for instance in `decodePayReq` to check if the argument is present, and also give it a name